src/map/packets/party_define.cpp

```diff
@@ -63,5 +64,16 @@ CPartyDefinePacket::CPartyDefinePacket(CParty* PParty)
 				i++;
 			}
 		}
+
+        if (PParty->GetLeader() != nullptr && PParty->GetLeader()->objtype == TYPE_PC) {
+            CCharEntity* leader = (CCharEntity*)PParty->GetLeader();
+            for (auto trust : leader->PTrusts) {
+                ref<uint32>(12 * i + (0x08)) = trust->id;
+                ref<uint16>(12 * i + (0x0C)) = trust->targid;
+                ref<uint16>(12 * i + (0x0E)) = 0;
+                ref<uint16>(12 * i + (0x10)) = trust->getZone();
+                i++;
+            }
+        }
 	}
 }
```

Comment:
i realise the tabs here are from previous commits, but can you tabs -> 4 spaces this entire file while youre working on it since its small enough and we can ignore whitespace diff anyways

Reply:
Done. https://github.com/DarkstarProject/darkstar/pull/4729/commits/3851ce978753a54d2977ae8e95806aa41e7e9036